index.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+// Key in annotations object that maps to an executor NPM module
+const ANNOTATION_EXECUTOR_TYPE = 'beta.screwdriver.cd/executor';
```

Comment:
The general idea was:

* `screwdriver.cd` is the domain that owns the annotation. 
* `beta` is the level of support/stability that the annotation has. 

But that depends heavily on how annotations are consumed and what the team ultimately decides on in screwdriver-cd/screwdriver#595